test-harness/specs/dynamic_xml_responses-AC-1.dynamic-response.oas2.txt

```diff
@@ -0,0 +1,31 @@
+====test====
+A dynamic XML response can be generated from the schema when "application/xml" Accept header is provided
+and Prism is started in dynamic mode.
+====spec====
+swagger: "2.0"
+paths:
+  /todos:
+    get:
+      produces:
+        - application/xml
+      responses:
+        200:
+          description: Get Todo Items
+          schema:
+            type: object
+            properties:
+              title:
+                type: string
+              description:
+                type: string
+              priority:
+                type: integer
+====server====
+mock -d -p 4010
+====command====
+curl -i http://localhost:4010/todos -H "accept: application/xml"
+====expect-loose====
+HTTP/1.1 200 OK
+
+<title>enim</title><description>id dolore</description><priority>90439782</priority>
```

Comment:
Hey, correct me if I'm wrong but I think this is not a valid XML. XMLs need a root element, right?
Should we wrap it with empty root? `<></>` or `<response></response>`?

Reply:
This is fixed in Prism 3.1 — rebase it with the current master and you should get this.

Reply:
`<></>` might be valid JSX but that's not a thing in XML.